AdsHelper/src/main/java/com/sofit/adshelper/core/AdsHelper.kt

```diff
@@ -123,19 +126,23 @@ object AdsHelper {
     }
 
     @JvmStatic
-    fun showAdMobBanner(activity: Activity, rLayout: RelativeLayout) {
-        if (isUserVerified) {
-            AdMobBanner.showAdMobBanner(activity, rLayout)
+    fun showNativeBannerAds(activity: Activity, rLayout: RelativeLayout, adNetwork: AdNetwork) {
+        when (adNetwork) {
+            AdNetwork.AdMob -> {
+                if (isUserVerified) {
```

Comment:
This can be moved outside the `when` method's scope if you need it for all its branches.